src/input.rs

```diff
@@ -13,7 +13,7 @@ use crate::error::*;
 /// This tells bat how to refer to the input.
 #[derive(Clone)]
 pub struct InputDescription {
-    pub(crate) name: String,
+    pub name: String,
```

Comment:
Same for this. It shouldn't be directly accessible as part of the API, and we already have `my_input_description.title()` as a way to get the appropriate title for the file (or name if no title is specified).